Models/HumanDetection/HumanDetector.py

```diff
@@ -205,7 +205,7 @@ def display_training_graphs(src_location: str):
 
 if __name__ == "__main__":
```

Comment:
The training looks pretty reasonable; for the demo we would probably have it pretrained and need a test/inference method to be called, right? Is that stored somewhere else or not written yet?

For the accuracy we can always just find some images that it outputs the correct result on, and make sure to use those in the demo to ensure that it exhibits the correct behaviour.

Reply:
I have a trained model stored on my laptop, I can just give the weights to whoever is running the demo on a usb stick. 

testing/inference is all stored here, you can see how it's done in app/main.py

That's a good idea, I just have human detection disabled by default in main.py but if we want to show it then yeah lets do that :)